.eleventy.js

```diff
@@ -82,6 +82,7 @@ module.exports = function(eleventyConfig) {
 
     // make global accessible in src/_includes/layouts/base.njk for loading of PH scripts
     eleventyConfig.addGlobalData('POSTHOG_APIKEY', () => process.env.POSTHOG_APIKEY || '' )
+    eleventyConfig.addGlobalData('DEV_MODE', () => !DEV_MODE && !DEV_MODE_POSTS)
```

Comment:
wouldn't it make more sense
```suggestion
    eleventyConfig.addGlobalData('DEV_MODE', () => DEV_MODE && DEV_MODE_POSTS)
```

Reply:
Or, instead of calling it `dev_mode`, make it `prod_mode`? And update de if in `base.njk`